### PR TITLE
add playback speed controls

### PR DESCRIPTION
**Pull request by vapier**

### CODE REVIEW

File: webui-page/webui.js [hunk 1]
```diff
@@ -19,6 +19,7 @@ function send(command, param){
   request.open("post", path);
 
   request.send(null);
+  setTimeout(status, 500);
```
Comment (open-dynaMIX) [inline webui-page/webui.js:22]: I prefer leaving the `setTimeout()` below at `1000`.

File: webui-page/webui.js [hunk 2]
```diff
@@ -491,7 +509,7 @@ function setupNotification() {
 }
 
 status();
-setInterval(function(){status();}, 1000);
+setInterval(function(){status();}, 10000);
```
Comment (open-dynaMIX) [inline webui-page/webui.js:512]: With a timeout of `1000`, one can see the seconds counting in the webui. IMO this gives a much more "real-time" feel of the app. Please revert.

File: webui.lua [hunk 3]
```diff
@@ -603,7 +625,8 @@ if passwd ~= 1 then
             .. concatkeys(servers, ':' .. options.port .. ' and ')
             .. ":" .. options.port
     )
-    message = function() mp.osd_message(MSG_PREFIX .. startup_msg, 5) end
+--    message = function() mp.osd_message(MSG_PREFIX .. startup_msg, 5) end
+    message = function() end
```
Comment (open-dynaMIX) [inline webui.lua:629]: Ok, you don't like the startup message :laughing: .

We could have an option to toggle them off. But that would be something for another PR. Please revert.

File: webui.lua [hunk 4]
```diff
@@ -593,7 +615,7 @@ local servers = init_servers()
 if passwd ~= 1 then
   if next(servers) == nil then
     error_msg = "Error: Couldn't spawn server on port " .. options.port
-    message = function() mp.msg.error(error_msg); mp.osd_message(MSG_PREFIX .. error_msg, 5) end
+    message = function() mp.msg.error(error_msg); mp.osd_message(MSG_PREFIX .. error_msg, 2) end
```
Comment (open-dynaMIX) [inline webui.lua:618]: See my other comment about the startup message.

File: webui-page/webui.js (hunk 1 above)
Comment (vapier) [reply webui-page/webui.js]: sorry, didn't mean to include these local hacks in this PR

File: webui.lua (hunk 3 above)
Comment (vapier) [reply webui.lua]: sorry, didn't mean to include these local hacks in this PR

File: webui.lua (hunk 4 above)
Comment (vapier) [reply webui.lua]: sorry, didn't mean to include these local hacks in this PR

**Comment by vapier** [comment]
made all the requested changes

**Comment by open-dynaMIX** [comment]
Thanks a lot @vapier! :tada:
